comment_scan.rb

```diff
@@ -27,7 +27,7 @@
 cb.login(cookie_file: 'cookies.yml')
 cb.say("_Starting at rev #{`git rev-parse --short HEAD`.chop} on branch #{`git rev-parse --abbrev-ref HEAD`.chop} (#{`git log -1 --pretty=%B`.gsub("\n", '')})_", HQ_ROOM_ID)
 cb.join_room HQ_ROOM_ID
-cb.join_rooms ROOMS #THIS IS THE PROBLEM
+cb.join_rooms ROOMS # THIS IS THE PROBLEM
```

Comment:
This comment can prolly go away unless we want to leave it because it's kinda funny.

Reply:
Haha looks like you added this one more than 9 months ago! I figured there was probably some reason and left it, but now I'm leaving it on purpose :P